esmf_regrid/tests/unit/experimental/unstructured_scheme/test__mesh_to_MeshInfo.py

```diff
@@ -0,0 +1,55 @@
+"""Unit tests for :func:`esmf_regrid.experimental.unstructured_scheme._mesh_to_MeshInfo`."""
+
+from iris.coords import AuxCoord
+from iris.experimental.ugrid import Connectivity, Mesh
+import numpy as np
+from numpy import ma
+import scipy.sparse
+
+from esmf_regrid.esmf_regridder import Regridder
+from esmf_regrid.experimental.unstructured_scheme import _mesh_to_MeshInfo
+
+
+def _example_mesh():
+    """Generate a global mesh with a square pyramid topology."""
+    fnc_array = [
```

Comment:
Maybe add a comment in here:
`# construct face-node-connectivity (fnc)`

Reply:
Agree, a bit of explanation of what is being made here would be helpful

Reply:
ASCII diagram possible?!

Reply:
great! excellent artistic skills 🎨